universe/node/api_auth.go

```diff
@@ -223,19 +223,18 @@ func (n *Node) apiGuestToken(c *gin.Context) {
 		return
 	}
 
-	outUser := dto.User{
-		ID:        userEntry.UserID.String(),
-		Name:      *userEntry.Profile.Name,
-		CreatedAt: time.Now().Format(time.RFC3339),
-		JWTToken:  &token,
-		IsGuest:   true,
-	}
-
-	if userEntry.UserTypeID != nil {
-		outUser.UserTypeID = userEntry.UserTypeID.String()
+	name := userEntry.UserID.String()
+	if userEntry.Profile.Name != nil {
+		name = *userEntry.Profile.Name
 	}
-	if userEntry.UpdatedAt != nil {
-		outUser.UpdatedAt = utils.GetPTR(userEntry.UpdatedAt.Format(time.RFC3339))
+	outUser := dto.User{
```

Comment:
I just meant this for instance, looks like you added some extra fields. Which i thought might not have to do with the naming refactor itself, since you added this as intended all is good.

Reply:
look, I just moved the field into `outUser` itself